circle_ci_environ.py

```diff
@@ -41,6 +41,7 @@
     'CI_PULL_REQUEST',
     'CI_PULL_REQUESTS',
     'CI_REPORTS',
+    'YOLO_ENV',
```

Comment:
@daspecster Remove this, squash into a single commit and then update the PR name from "Yolo environment" so we can merge?